src/components/Explore/interfaces.ts

```diff
@@ -21,19 +21,26 @@ export interface Series {
   params?: SeriesParams;
 }
 
+/**
+ * Indices assigned to each enumerated explore metric needs to match
+ * the order of the dropdown items in EXPLORE_METRICS in Explore/utils.ts.
+ *
+ * So if we change the ordering of items in EXPLORE_METRIC,
```

Comment:
EXPLORE_METRICS*

id be explicit and say `EXPLORE_METRICS in Explore/utils.ts`